src/app/components/compliance-plot/compliance-plot.component.ts

```diff
@@ -21,17 +19,39 @@ import * as complianceAction from '../../shared/store/compliance/compliance.acti
 export class CompliancePlotComponent implements OnInit {
 
   @Input() studyId
+  @Input() timeHoles = true
   data$: Observable<any>
   isComplianceLoaded$: Observable<boolean>
+  payload: any
+
+  // TODO: Part of config
+  complianceKeys =
```

Comment:
Yes. I've added it to the config with a slightly different structure for consistency: 
https://radar-dashboard.firebaseio.com/config.json